src/Product/Image/Assignor.php

```diff
@@ -0,0 +1,96 @@
+<?php
```

Comment:
Do we need an entirely new class just to change some variables on an Image object? Could this not be a method on Image/Edit? (I know this class doesn't exist but "Assignor" isn't used anywhere else, having an "Edit class would make more sense no?)